### DISCUSSION TITLE
Looking to center the PDF in the PDF viewer

### DISCUSSION THREAD

**Discussion by wjmack** [stephanrauh/ngx-extended-pdf-viewer#1468] [Q&A] [answered]
Wondering if there is an attribute for the PDF viewer to center the pages in the viewer? Right now they are left-aligned.

Thanks!

**Comment by stephanrauh** [comment, marked as answer]
They're only left-aligned in single-page mode. This is a bug, but I'm not sure I want to fix it because it'd break compatibility. Maybe you're right, I should add an attribute. In the meantime, you can use a global CSS rule in your `styles.css` to fix the bug:

```css
ngx-extended-pdf-viewer .pdfViewer.singlePageView .page {
  margin-right: auto;
  margin-left: auto;
```

Maybe you'll have to add `!important`, but I don't think so.

**Reply by wjmack** [reply]
That worked great, thank you. And yes, I haven't used multi-page mode in a while so I had forgotten that it was originally centered. I appreciate your help.
